### PR TITLE
ICDS: More Report Tests and Changes

### PR DESCRIPTION
**Pull request by sheelio**
This should be the last set of tests for the ICDS report data sources.   

@czue 
Feel free to review commit-by-commit if you'd like.  You can ignore the first commit.

### CODE REVIEW

File: custom/icds_reports/ucr/expressions.py [hunk 1]
```diff
@@ -846,3 +878,11 @@ def child_valid_in_month(spec, context):
...
     }
     return ExpressionFactory.from_spec(spec, context)
+
+
+def get_child_cases(spec, context):
+    wrapped = GetChildCasesSpec.wrap(spec)
+    wrapped.configure(
+        case_id_expression=ExpressionFactory.from_spec(wrapped.case_id_expression, context),
+    )
+    return wrapped
```
Comment (dimagimon) [inline custom/icds_reports/ucr/expressions.py]: W292 no newline at end of file

File: custom/icds_reports/ucr/tests/expression_tests.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
+import uuid
+from datetime import date, datetime
```
Comment (dimagimon) [inline custom/icds_reports/ucr/tests/expression_tests.py]: F401 'date' imported but unused

File: custom/icds_reports/ucr/tests/test_child_health_monthly.py [hunk 3]
```diff
@@ -1282,28 +1425,155 @@ def test_cf(self):
...
+            form_date=datetime(2015, 10, 10),
+            case_id='m-' + case_id,
+            counsel_immediate_bf='yes',
+        )
+
+        cases = [(0, [('counsel_immediate_breastfeeding', 0)]),
+                 (1, [('counsel_immediate_breastfeeding', 1)]),
+                 (2, [('counsel_immediate_breastfeeding', 0)]),
+                 ]
+        self._run_iterative_monthly_test(case_id=case_id, cases=cases)
```
Comment (dimagimon) [inline custom/icds_reports/ucr/tests/test_child_health_monthly.py]: W292 no newline at end of file

**Review by czue — APPROVED** [review]
well done on the tests

**Pull request merged**